README.md

```diff
@@ -60,8 +60,32 @@ Most of the components within this project use Kotlin coroutines and `Flow` to d
  
 ## Publishing
 
-The libraries are not published on any hosted repository yet, but can be published to your local maven repo  
- 
+Kaluga is published on Bintray and available in JCenter. Also it can be published to your local maven repo.
```

Comment:
Unfortunatly not everything is on jcenter (date picker at least is missing), so best to give instructions how to add our bintray repo for now.